changelog/unreleased/pr-17075.toml

```diff
@@ -0,0 +1,4 @@
+type = "c"
+message = "Updated io.jsonwebtoken:jjwt-api from 0.11.5 to 0.12.3"
```

Comment:
We don't need a changelog entry for the update. The update doesn't change anything from the user's perspective.